src/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixer.php

```diff
@@ -73,6 +73,10 @@
 
     /**
      * {@inheritdoc}
+     *
+     * @SuppressWarnings(PHPMD.SwitchStatement)
+     *
+     * @todo drop warning suppression
```

Comment:
`@todo` later? Or did you forgot to handle it in this PR?

Reply:
@todo for non 2.2 branch,
actually, it's already resolved in #2304, just didn't wanted to backport it as it's not just style that changed but also a logic (in yet another PR)